src/features/profile/pages/DeleteProfile/DeleteProfileAccountNotDeletable.tsx

```diff
@@ -0,0 +1,87 @@
+import { useNavigation } from '@react-navigation/native'
+import React, { FC } from 'react'
+import styled from 'styled-components/native'
+
+import { UseNavigationType } from 'features/navigation/RootNavigator/types'
+import { getTabNavConfig } from 'features/navigation/TabBar/helpers'
+import { env } from 'libs/environment'
+import { ButtonInsideText } from 'ui/components/buttons/buttonInsideText/ButtonInsideText'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { ButtonTertiaryBlack } from 'ui/components/buttons/ButtonTertiaryBlack'
+import { ExternalTouchableLink } from 'ui/components/touchableLink/ExternalTouchableLink'
+import { ViewGap } from 'ui/components/ViewGap/ViewGap'
+import { GenericInfoPageWhite } from 'ui/pages/GenericInfoPageWhite'
+import { BellFilled } from 'ui/svg/icons/BellFilled'
+import { BicolorError } from 'ui/svg/icons/BicolorError'
+import { ExternalSiteFilled } from 'ui/svg/icons/ExternalSiteFilled'
+import { getSpacing, Typo, TypoDS } from 'ui/theme'
+
+export const DeleteProfileAccountNotDeletable: FC = () => {
+  const { navigate } = useNavigation<UseNavigationType>()
+
+  const navigateToProfile = () => navigate(...getTabNavConfig('Profile'))
+
+  const navigateToNotifications = () => navigate('NotificationsSettings')
+
+  return (
+    <GenericInfoPageWhite
+      headerGoBack
+      separateIconFromTitle={false}
+      icon={BicolorError}
+      titleComponent={Typo.Title2}
+      title="Nous ne pouvons pas encore supprimer ton compte">
+      <Content>
+        <ViewGap gap={6}>
+          <TypoDS.BodyS>
+            Pour éviter les cas de fraude et te permettre d’accéder à ton crédit, il est nécessaire
+            de garder ton compte pour l’instant.
+          </TypoDS.BodyS>
+          <TypoDS.BodyS>
+            À tes 21 ans, ton compte pourra être supprimé et tu pourras faire une demande pour
+            anonymiser tes données. Tu peux en savoir plus en{' '}
+            <ExternalTouchableLink
+              as={StyledButtonInsideText}
+              wording="consultant cette page"
+              icon={ExternalSiteFilled}
+              externalNav={{ url: env.FAQ_LINK_PERSONAL_DATA }}
+            />
+            .
+          </TypoDS.BodyS>
+          <TypoDS.BodyS>
+            Pour ne plus recevoir de communications du pass Culture, tu peux{' '}
+            <StyledBoldText>désactiver tes notifications</StyledBoldText>.
+          </TypoDS.BodyS>
```

Comment:
C'est plus lisible comme ça 

```
import { SPACE } from 'ui/theme/constants'
```

```suggestion
          <TypoDS.BodyS>
            Pour ne plus recevoir de communications du pass Culture, tu peux
            {SPACE}
            <StyledBoldText>désactiver tes notifications</StyledBoldText>.
          </TypoDS.BodyS>
```